contents/handbook/people/team-structure/app-east.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Team App East
+sidebar: Handbook
+showTitle: true
+hideAnchor: true
+---
+
+## People
+
+- Marius Andra (Team lead)
+- Ben White (Engineer)
+- Paul D'Ambra (Engineer)
+- Michael Matloka (Engineer)
+- Emanuele Capparelli (Growth Engineer)
+
+## Mission
+
+Makers everywhere get better at building products because of PostHog
+
+## Q3 2022 Goals
+* **Proposed Objective:**
+    * Increase the number of successful new organizations on PostHog
```

Comment:
Can we qualify what "successful" means here? I'm anticipating it contains some key milestones, but some clarification would help for sure. 
- Ingested events
- Invited other project / org members
- Discovered learning(s)
- Entered payment?
- More?

Reply:
The objective should be slightly ambiguous, since we might not have a single good way of measuring success, then we can use the key results to clarify what success actually looks like in terms of how we would measure it.